test/public/logs/overview.test.js

```diff
@@ -70,16 +72,16 @@ module.exports = () => {
 
     it('Should display the correct items counter at the bottom of the page', async () => {
         await goToPage(page, 'log-overview');
-        await waitForTimeout(100);
+        await page.waitForSelector('#firstRowIndex');
 
         expect(await page.$eval('#firstRowIndex', (element) => parseInt(element.innerText, 10))).to.equal(1);
         expect(await page.$eval('#lastRowIndex', (element) => parseInt(element.innerText, 10))).to.equal(10);
-        expect(await page.$eval('#totalRowsCount', (element) => parseInt(element.innerText, 10))).to.equal(138);
+        expect(await page.$eval('#totalRowsCount', (element) => parseInt(element.innerText, 10))).to.equal(119);
     });
 
     it('Should have balloon on title, tags and runs columns', async () => {
         await goToPage(page, 'log-overview');
-        await waitForTimeout(100);
+        await page.waitForFunction(() => document.querySelectorAll('table tbody tr').length === 10, { timeout: 1500 });
```

Comment:
Here, I would simply wait for the table to have at least one row that is not loading row, to make tests a bit more robust (no need for length here)
```suggestion
        await page.waitForSelector('tbody tr:not(.loading-row)');
```

Reply:
Done